geomstats/learning/frechet_mean.py

```diff
@@ -253,9 +253,7 @@ def _adaptive_gradient_descent(points,
 
     sum_weights = gs.sum(weights)
 
-    n_init = len(init_points)
-
-    if n_init == 0:
+    if init_points is None:
```

Comment:
Will this also works when `init_points` is given as `[]` by the user?

Reply:
According to DeepSource setting [] as default value is not recommended. There is no other reason a user would pass []?

Reply:
In fact even if different init_points are passed, only the first one is used